src/transformations/gridshift.cpp

```diff
@@ -372,6 +372,7 @@ PJ_XYZ gridshiftData::grid_interpolate(PJ_CONTEXT *ctx, const std::string &type,
         gridInfo.idxSampleXYZ[2] = idxSampleZ;
         iterCache = m_cacheGridInfo.emplace(grid, std::move(gridInfo)).first;
     }
+    // cppcheck-suppress derefInvalidIteratorRedundantCheck
```

Comment:
How long is the effect of this suppression? the rest of the file?

Reply:
the line immediately below